### ISSUE TITLE
Theme is not applied to all Zendesk activities

### ISSUE THREAD

**Issue by juank-pa** [zendesk/sdk_unity_plugin#73]
I modified the styles.xml to make it inherit from the Dark theme and also modified the colors to match my company colors. But no matter how I changes things whenever I open the Help Center with the `ZDKHelpCenter.ShowHelpCenter()` method, all screens (Help Center home, ticket list, new ticket, etc) are shown with a white background including the ActionBar, except when I navigate to a specific article.

Steps to reproduce:
1. Modify XMLs to inherit dark theme.
   ```xml
   <resources>
     <style name="UnityTheme" parent="ZendeskSdkTheme.Dark">
        <item name="colorPrimary">@color/zendesk_color_primary</item>
        <item name="colorPrimaryDark">@color/zendesk_color_primary_dark</item>
        <item name="colorAccent">@color/zendesk_color_accent</item>
        <item name="android:windowBackground">@color/zendesk_background_color</item>

         <!--
            If you find that the accent color is not applied to the Dialog, you can
            uncomment this line, and customize AppTheme.Dialog to your requirements
        -->
        <!-- <item name="android:dialogTheme">@style/AppTheme.Dialog</item> -->
     </style>

     <style name="AppTheme.Dialog" parent="Theme.AppCompat.Light.Dialog">
        <item name="colorPrimary">@color/zendesk_color_primary</item>
     </style>
   </resources>
   ```
1. Update colors in `colors.xml`.
1. Recompile the plugin with `./gradlew build`
1. Ensure I set the application theme to `UnityTheme`
1. Build and run the Android apk

For some reason only when navigating to an article from the help desk home shows an activity properly themed. This makes it look completely awkward.

I tried many different approaches like:

* Changed the name of the theme in the Android library reflecting changes in Unity.
* Set the theme to all the Zendesk activities manually.
* Uncommenting the "android:dialogTheme" item and changing "AppTheme.Dialog" to inherit "Theme.AppCompat.Dialog" instead.

None of the work. I'm using Unity 2017.2.1p4 targeting Android API level 26. I did not user the support libraries provided by the plugin because I'm using v27.x.x ones but in theory they should be backward compatible.

I append captures (only last one gets the theme):
<img src="https://user-images.githubusercontent.com/5060527/38908902-e4d27a6a-4287-11e8-9ec0-309a014ab032.png" width="150" /> <img src="https://user-images.githubusercontent.com/5060527/38908905-e50ca488-4287-11e8-847d-9c950647ac0f.png" width="150" /> <img src="https://user-images.githubusercontent.com/5060527/38908904-e4f0ab48-4287-11e8-816f-7da399e7129d.png" width="150" /> <img src="https://user-images.githubusercontent.com/5060527/38908901-e4ba3662-4287-11e8-949a-5a89ebba93ce.png" width="150" />

**Comment by brendan-fahy** [comment]
Hi @juank-pa, a couple of us have tried to reproduce this here and haven't been able to. We're seeing the theme applied to all Activities. Can you share a couple of relevant portions of the android-plugin `build.gradle` with us so we can see exactly what we're working with?
Specifically, if you've changed compile/target SDK versions, show us what you have instead of:
```
android {
    compileSdkVersion 25
    buildToolsVersion "25.0.2"

    defaultConfig {
        minSdkVersion 15
        targetSdkVersion 25
        versionName "1.11.0.1"
        vectorDrawables.useSupportLibrary = true
    }
    lintOptions {
        abortOnError false
    }
}
```

Can you also share your imports of the 27.x.x versions of the support libraries? 
Also, to clarify, have you customised the CSS for the article? I'd be surprised to see a blue background on it if you hadn't done that.

**Comment by juank-pa** [comment]
Yes I also customized the CSS. I forgot to mention that things fixed up a little when removing comments from 
```xml
<!-- <item name="android:dialogTheme">@style/AppTheme.Dialog</item> -->
```

**Comment by juank-pa** [comment]
After that screens started to theme correctly but then I started having other styling issues, because some zendesk sdk elements  and screens were not using the colors coming from the theme but specific burned out colors. For instance the request and comments screens even when having the navbar correctly themed the list was itself gray and list items were black with white text. I tried to replace those styles in my own project res files but didn't work. Also I was having problems with belvedere icons that were always dark (burned out in code).

So I had to change the build process to make it not only remove the css files but also the zendesk SDK values.xml. I copied and pasted that file myself and customized it in my project. I did the same to remove the belvedere icons and replace them with white versions.

In summary I fixed most of it. So we could call it fixed. If you want a reference of what I did you can see it in this fork. Thanks anyway ;)

**Comment by brendan-fahy** [comment]
Hey @juank-pa, glad to hear you got it sorted. That's very interesting feedback, the lengths you had to go to, to get this sorted. It's not something we've had reported to us before. I had a look at your fork but didn't see any new commits there, will you be adding them soon? It would be great to see the changes 👍

**Comment by juank-pa** [comment]
Sorry, it was still in a branch
https://github.com/juank-pa/zen_desk_unity_plugin/tree/customized_zendesk

**Comment by brendan-fahy** [comment]
Thanks!

**Comment by mohsinbuttdoduae** [comment]
User this in your menifest file 

 <activity android:name="zendesk.messaging.MessagingActivity"
            android:theme="@style/ZendDeskTheme" />
        <activity android:name="zendesk.answerbot.AnswerBotArticleActivity"
            android:theme="@style/ZendDeskTheme" />
        <activity android:name="zendesk.support.requestlist.RequestListActivity"
            android:theme="@style/ZendDeskTheme" />
        <activity android:name="zendesk.support.request.RequestActivity"
            android:theme="@style/ZendDeskTheme" />
        <activity android:name="zendesk.support.guide.ViewArticleActivity"
            android:theme="@style/ZendDeskTheme" />


and add this in your Style


<style name="ZendDeskTheme" parent="ZendeskSdkTheme.Light">
        <item name="colorPrimary">@color/colorPrimary_zendesk</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark_zendesk</item>
        <item name="colorAccent">@color/colorAccent_zendesk</item>
        <item name="myLinkColor">@color/colorAccent_zendesk</item>
        <item name="android:windowAnimationStyle">@style/CustomActivityAnimation</item>
    </style>
